Sources/OTel/OTelAPI/OTel+Bootstrap.swift

```diff
@@ -124,32 +124,57 @@ extension OTel {
         var services: [Service] = []
 
         if configuration.logs.enabled {
-            try services.append(bootstrapLogs(configuration: configuration))
+            try services.append(bootstrapLogs(configuration: configuration, logger: logger))
         }
         if configuration.metrics.enabled {
-            try services.append(bootstrapMetrics(configuration: configuration))
+            try services.append(bootstrapMetrics(configuration: configuration, logger: logger))
         }
         if configuration.traces.enabled {
-            try services.append(bootstrapTraces(configuration: configuration))
+            try services.append(bootstrapTraces(configuration: configuration, logger: logger))
         }
 
         return ServiceGroup(services: services, logger: logger)
     }
 
-    internal static func bootstrapTraces(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapTraces(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeTracingBackend(configuration: configuration)
         InstrumentationSystem.bootstrap(backend.factory)
         return backend.service
     }
 
-    internal static func bootstrapMetrics(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapMetrics(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeMetricsBackend(configuration: configuration)
         MetricsSystem.bootstrap(backend.factory)
         return backend.service
     }
 
-    internal static func bootstrapLogs(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapLogs(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeLoggingBackend(configuration: configuration)
+        if configuration.logs.exporter.backing != .console {
+            logger.info(
+                """
+
+                Bootstapping logging system
+                ---
+                Only diagnostic logging will use the console logger.
```

Comment:
```suggestion
                Bootstapping logging system using Swift OTel
                ---
                Only diagnostic logging emitted by Swift OTel will use the console logger.
```

Feel free to play with the wording, but for a new person to a codebase, it might not be immediately obvious which packages is emitting this message in their executable, so adding the name here will make it easier to Google if they get stuck, and know which package to look up more docs for.

Reply:
OK, done 🙏 https://github.com/swift-otel/swift-otel/pull/306/commits/d745e024731eec1e9f4ccdcc07ed7b8e0ab8e6aa